packages/bot-web-ui/src/components/quick-strategy/quick-strategy.jsx

```diff
@@ -167,217 +167,434 @@ const QuickStrategyForm = ({
                                     )}
                                 </Field>
                             </div>
-                            <div className='quick-strategy__form-row quick-strategy__form-row--multiple'>
-                                <Field name='quick-strategy__duration-unit'>
-                                    {({ field }) => (
-                                        <>
-                                            {is_mobile ? (
-                                                <SelectNative
-                                                    list_items={duration_unit_dropdown}
-                                                    value={selected_duration_unit.value}
-                                                    label={localize('Duration unit')}
-                                                    should_show_empty_option={false}
+                            {is_mobile ? (
```

Comment:
We have `@mixin mobile`. Is it possible to use that in scss instead of the `is_mobile` check here?

Reply:
@nijil-binary unfortunately no as we need to change the layout here, not styles. `is_mobile` is common used in this file, so I think we shall keep consistent

Reply:
I believe the styles are changed using this className `quick-strategy__form-row--multiple`, so why not use `@include mobile` in `quick-strategy__form-row` instead? It'd help reduce unnecessary duplication of code.

Reply:
no, each input field is moved to a separate container as first 2 inputs, `quick-strategy__form-row--multiple` is removed from containers where only 1 input inside. Main point of this code is to change the structure of inputs and containers for mobile. I believe we don't need the style change here